templates/lightning/layout.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
if this file is not used then get rid of it

Reply:
removed this page